Doc/library/ssl.rst

```diff
@@ -934,6 +934,12 @@ Constants
 
    .. versionadded:: 3.13
 
+.. data:: HAS_PHA
+
+   Whether the OpenSSL library has built-in support for TLS post-handshake auth (PHA).
```

Comment:
```suggestion
   Whether the OpenSSL library has built-in support for TLS-PHA.
```

We don't specify what TLS-PSK is, so I don't think we need to specify what PHA is.